src/box/lua/net_box.lua

```diff
@@ -155,10 +155,11 @@ local function create_transport(host, port, user, password, callback,
     if user ~= nil and password == nil then password = '' end
 
     -- Current state machine's state.
-    local state            = 'initial'
+    local state             = 'initial'
     local last_errno
     local last_error
-    local state_cond       = fiber.cond() -- signaled when the state changes
+    local state_cond        = fiber.cond() -- signaled when the state changes
+    local on_send_buf_empty = fiber.cond() -- signaled when send_buf:size() == 0
```

Comment:
Let's move it after send_buf definition so that the comment doesn't refer to the future.

Reply:
fixed